docs/administration/teams-users/team-based-access.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Team-based access
+id: team-based-access
+description: Manage access by team
+tags: []
+---
+
+In Prophecy, access control operates through team associations with resources. Projects and fabrics are the fundamental resources to which teams are assigned. Users who belong to a team can access all projects and fabrics assigned to that team.
+
+This guide provides comprehensive best practices for implementing and managing team-based access to optimize collaboration while maintaining appropriate security boundaries.
+
+:::note
+Team admins have special privileges for managing team membership, but the core access model remains focused on the team-to-resource relationship. Each team can be assigned to many projects and fabrics, but importantly, each project and fabric can only be assigned to one team.
+:::
```

Comment:
I would add a small diagram here with only three boxes: teams, fabrics, and projects. Show the dots and arrows accordingly 😄